views/update/update-beneficial-owner-type.html

```diff
@@ -94,7 +94,10 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
     <form method="post">
       {% include "includes/csrf_token.html" %}
       {% include "includes/beneficial-owner-type-add-radios.html" %}
-
+      {% if update.change_bo_relevant_period === "CHANGE_BO_RELEVANT_PERIOD" %}
+      <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
```

Comment:
Can you add some unit tests to `benifical.owner.type.spec` to check these extra radio buttons are rendered. you can pass the mock `UPDATE_OBJECT_MOCK_RELEVANT_PERIOD_CHANGE` in `session.mock` to mock the first checkbox being ticked.